api/metric/api_test.go

```diff
@@ -538,15 +480,15 @@ func checkBatches(t *testing.T, ctx context.Context, labels LabelSet, meter *moc
 		}
 		for j := 0; j < minMLen; j++ {
 			measurement := got.measurements[j]
-			if measurement.Descriptor != descriptor {
-				d := func(d *Descriptor) string {
-					return fmt.Sprintf("(ptr: %p, descriptor %#v)", d, d)
+			if measurement.instrument != ourInstrument {
+				d := func(i *mockInstrument) string {
+					return fmt.Sprintf("(ptr: %p, descriptor %#v)", i, i)
```

Comment:
nit: s/descriptor/instrument/

Reply:
Oops. Fixed this here and in other places too.